english/chopper/get-to-it.md

```diff
@@ -0,0 +1,3 @@
+The bananas were hot while the tea was cold. What craziness is this? Give me a current bun and be done with it all, I thought. At least I don't have to get to the [chopper](http://s2.quickmeme.com/img/98/982c902d16cad216960055a63f97c0f776ef6745d52c9713a2c5e8c5a07ce115.jpg)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.